### PR TITLE
Fix the CMakeLists.txt to sort the cpp files in alphabetical order

### PR DESCRIPTION
**Pull request by liujiayi771**
Sorting the .cpp files alphabetically can improve readability and maintainability of the CMake configuration.

### CODE REVIEW

**Comment by liujiayi771** [comment]
@mbasmanova Could you help to review?

**Comment by liujiayi771** [comment]
Related issue https://github.com/facebookincubator/velox/issues/9169.

**Comment by liujiayi771** [comment]
@mbasmanova I change the order of cpp files in `velox/expression/tests/CMakeLists.txt`, `ArrayViewTest` and `ArrayWriterTest` are executed before `CustomTypeTest`, the customer UDT registered in `ArrayViewTest` and `ArrayWriterTest` will be in the return values of `getCustomTypeNames()`.

**Review by mbasmanova — APPROVED** [review]
Thank you for the cleanup.

I wonder if it is possible to automate this. It would be nice to extend `make format-fix` command to ensure file names appear in alphabetical order in cmake files.

File: velox/expression/tests/CustomTypeTest.cpp [hunk 1]
```diff
@@ -227,6 +229,8 @@ TEST_F(CustomTypeTest, getCustomTypeNames) {
           "JSON",
           "HYPERLOGLOG",
           "TIMESTAMP WITH TIME ZONE",
+          "UDT2",
+          "MATERIALIZETESTUDT",
```
Comment (mbasmanova) [inline velox/expression/tests/CustomTypeTest.cpp:233]: Where are these from?

Comment (mbasmanova) [reply velox/expression/tests/CustomTypeTest.cpp:233]: We should not hard-code custom types registered in other tests like this. It is fragile and will break again if another test is added to register some new types.

Comment (liujiayi771) [reply velox/expression/tests/CustomTypeTest.cpp:233]: @mbasmanova Can we call `unregisterCustomType` after the use of the custom type?

**Comment by liujiayi771** [comment]
@mbasmanova I added code to unregister these UDTs after using them in tests.

File: velox/expression/tests/CustomTypeTest.cpp (hunk 1 above)
Comment (mbasmanova) [reply velox/expression/tests/CustomTypeTest.cpp]: @liujiayi771 I see. Let's update ArrayViewTest and ArrayWriterTest to unregister custom types they use. Let's make these changes in a separate PR though.

File: velox/expression/tests/ArrayViewTest.cpp [hunk 2]
```diff
@@ -589,6 +589,8 @@ TEST_F(NullableArrayViewTest, materializeArrayOfCustomTypes) {
   std::vector<std::optional<UDT>> expected = {
       UDT{0}, std::nullopt, UDT{1}, UDT{2}};
   ASSERT_EQ(array, expected);
+
+  UDTTypeRegistrar::unregisterType();
```
Comment (mbasmanova) [inline velox/expression/tests/ArrayViewTest.cpp:593]: Thanks. Let's extract these changes into a separate PR. Let's also make sure that this code is called even if test fails. Maybe use folly::makeGuard from folly/ScopeGuard.h

File: velox/expression/tests/CustomTypeTest.cpp (hunk 1 above)
Comment (liujiayi771) [reply velox/expression/tests/CustomTypeTest.cpp]: @mbasmanova Should this current PR add the UDTs they registered to `CustomTypeTest` and then fix it in a separate PR?

Comment (mbasmanova) [reply velox/expression/tests/CustomTypeTest.cpp:233]: I suggest to create a PR to fix ArrayViewTest and ArrayWriterTest and land that first. Then rebase this PR and land it.

Comment (liujiayi771) [reply velox/expression/tests/CustomTypeTest.cpp:233]: OK. I will do this right now.

**Review by Yuhta — APPROVED** [review]

**Pull request closed without merging**